spec/requests/security_activity_spec.rb

```diff
@@ -154,13 +167,24 @@
     expect_event SecurityActivity::EMAIL_CHANGED, notes: "to #{user.email}"
   end
 
-  def expect_event(event, application: nil, factor: nil, notes: nil, analytics: nil)
-    events = user.security_activities.of_type(event)
-    events = events.where(oauth_application_id: application.id) if application
-    events = events.where(factor: factor) if factor
-    events = events.where(notes: notes) if notes
-    events = events.where(analytics: analytics) if analytics
+  def expect_event(event, **options)
+    options = default_expect_event_options.merge(options)
+    events = options[:user].security_activities.of_type(event)
+    events = events.where(oauth_application_id: options[:application].id) if options[:application]
+    events = events.where(factor: options[:factor]) if options[:factor]
+    events = events.where(notes: options[:notes]) if options[:notes]
+    events = events.where(analytics: options[:analytics]) if options[:analytics]
 
     expect(events.count).to_not eq(0)
   end
+
+  def default_expect_event_options
+    {
+      application: nil,
+      factor: nil,
+      notes: nil,
+      analytics: nil,
+      user: user,
+    }
+  end
```

Comment:
What's the benefit of this change?  `options[:user]` isn't used, and the `if`s mean that the default `nil` values are also unused.

Reply:
Fair enough, I've amended that to remove the default method.
https://github.com/alphagov/govuk-account-manager-prototype/pull/571/commits/9b261a12649b11d932ee2356c38f120f9de500cc